### PR TITLE
linearSearch programm added

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: algorithms/searchingAlgorithms/linearSearch/Makefile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+main.o: main.cpp
+	$(compiler) $(options) -c main.cpp
+
+linearSearch.o: linearSearch.cpp
+	$(compiler) $(options) -c linearSearch.cpp
+
+run: compile main 
+	./main
+
```
Comment (nvs-abhilash) [inline algorithms/searchingAlgorithms/linearSearch/Makefile:33]: remove extra line

File: algorithms/searchingAlgorithms/linearSearch/Makefile [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+clean:
+	rm -rf *.o main
```
Comment (nvs-abhilash) [inline algorithms/searchingAlgorithms/linearSearch/Makefile]: add extra line at end

File: algorithms/searchingAlgorithms/linearSearch/linearSearch.cpp [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+* along with this program.  If not, see <http://www.gnu.org/licenses/>
+*
+*/
+
+#include <iostream>
+#include "linearSearch.h"
+using namespace std;
+
+// function that searches for a number seqentially in an array.
+void linearSearch(int arry[], int length, int FindMe) {
```
Comment (nvs-abhilash) [inline algorithms/searchingAlgorithms/linearSearch/linearSearch.cpp]: give space between function name and parenthesis, also variable name is not according to coding guidlines

File: algorithms/searchingAlgorithms/binarySearch/main.cpp [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+*
+*/
+
+#include <iostream>
+#include "BinarySearch.h"
+#define MAX 10
+
+using namespace std;
+
+int main (void){
```
Comment (nvs-abhilash) [inline algorithms/searchingAlgorithms/binarySearch/main.cpp]: it should be
int main ()
{
}

File: algorithms/searchingAlgorithms/binarySearch/BinarySearch.cpp [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+*
+*/
+
+#include <iostream>
+#include "BinarySearch.h"
+using namespace std;
+
+// function that searches for a number using binary search algorithm.
+void BinarySearch(int arry[], int length, int FindMe) {
+    int startPos,endPos,midVal;
```
Comment (nvs-abhilash) [inline algorithms/searchingAlgorithms/binarySearch/BinarySearch.cpp]: add spaces after ,

**Review by nvs-abhilash — CHANGES_REQUESTED** [review]

**Comment by nvs-abhilash** [comment]
Squash all commits to one commit.

**Comment by ghost** [comment]
I have made the changes please verify them!

**Pull request closed without merging**